lib/logstash/util/plugin_version.rb

```diff
@@ -22,8 +22,14 @@ def initialize(*options)
 
     def self.find_version!(name)
       begin
-        specification = Gem::Specification.find_by_name(name)
-        new(specification.version)
+        #specification = Gem::Specification.find_by_name(name)
+        #new(specification.version)
+        build_from_spec(Gem::Specification.find_by_name(name))
+      rescue NoMethodError
```

Comment:
Why not just check for nil, instead of relying on exception to do the control flow?
```
undefined method `version' for nil:NilClass
```